OpenRA.Game/Game.cs

```diff
@@ -392,6 +393,17 @@ public static void InitializeMod(string mod, Arguments args)
 
 			JoinLocal();
 
+			try
+			{
+				discoverNat.Wait();
+			}
+			catch (Exception e)
+			{
+				Console.WriteLine("NAT discovery failed: {0}", e.Message);
+				Log.Write("nat", e.StackTrace);
```

Comment:
Fixed the NRE for the NAT discovery disabled case and added the whole exception to the log.